client/styles/style.css

```diff
@@ -35,12 +35,12 @@ a:active {
 }
 
 .search input {
-  width: 150px;
-  margin: 0 0 30px 0;
+  width: 100%;
```

Comment:
Nice! I was thinking this would be the a way to have it fit the container it's in regardless of size.